src/app/features/languageSlice.ts

```diff
@@ -0,0 +1,28 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
```

Comment:
changelog?

Reply:
Resolved in b4538f50cff6b58a39235132e5da1145b29ce061